spec/views/import_rdf/preview_import.html.erb_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe "import_rdf/preview_import.html.erb" do
+  let(:vocabns) { "http://opaquenamespace.org/ns/newthing" }
+  let(:form) { ImportForm.new }
+  let(:terms) { [Term.new(vocabns + "/1"), Term.new(vocabns + "/2"), Term.new(vocabns + "/3")] }
+
+  let(:vocabulary) do
```

Comment:
We're starting to do this a lot, I wonder if a FactoryGirl factory would be useful.